include/measurement_kit/neubot/dash.hpp

```diff
@@ -0,0 +1,26 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+#ifndef MEASUREMENT_KIT_NEUBOT_DASH_HPP
+#define MEASUREMENT_KIT_NEUBOT_DASH_HPP
+
+#include <functional>
+#include <measurement_kit/common.hpp>
+#include <measurement_kit/ext.hpp>
+
+namespace mk {
+namespace neubot {
+namespace dash {
+
+// Run a dash test
+void run(Settings settings, Callback<Error,  Var<nlohmann::json>> cb,
```

Comment:
Am I right in assuming that this step cannot run without knowing the server address? In such case, I'd refactor this function to receive a string first argument and move `settings` with optional arguments before `reactor` and initialized by default using `{}`. Also, given that `auth` is optional, I'd probably pass it using the settings.